src/react-native/xcode.ts

```diff
@@ -148,10 +164,38 @@ export function addSentryWithBundledScriptsToBundleShellScript(
     );
   }
 
+  if (!isLikelyPlainReactNativeScript && !isLikelyExpoScript) {
```

Comment:
I think, we should always show the copy-paste snippet when we fail to patch the script.

Even when the script is likely expo or likely plain RN the replace might fail.

Reply:
Good idea Krystof 👍 
Updated with 0b075ee1aa8dbad1d2cab747cca0fa1fc648720e to show the snippet when patching fails for expo and react native.